settings.talon

```diff
@@ -50,5 +50,33 @@ settings():
     # per-application basis.
     # user.context_sensitive_dictation = 1
 
+    # The update frequency used when moving a window continuously
+    user.win_move_frequency = "60ms"
+
+    # The target speed, in cm/sec, for continuous move operations
+    user.win_continuous_move_rate = 4.1
+
+    # When enabled, the 'Move/Resize Window' GUI will not be shown for continuous move operations
+    user.win_hide_move_gui = 0
+
+    # The update frequency used when resizing a window continuously
+    # note: setting this value too low may result in
+    user.win_resize_frequency = "60ms"
+
+    # The target speed, in cm/sec, for continuous resize operations
+    user.win_continuous_resize_rate = 4.0
+
+    # When enabled, the 'Move/Resize Window' GUI will not be shown for continuous resize operations
+    user.win_hide_resize_gui = 0
+
+    # How long to wait (in seconds) for talon to signal completion of window move/resize requests
+    user.win_set_queue_timeout = 0.2
+
+    # How many times to retry a timed out talon window move/resize request
+    user.win_set_retries = 1
+
+    # Whether to generate warnings for anomalous events.
+    user.win_verbose_warnings = 0
```

Comment:
This is a lot of settings, it would be good to double-check if there's some way to merge or remove some. But maybe there isn't!

Reply:
I split the move and resize settings because, on my system at least, the speeds differed even though the underlying code/mechanisms were the same.

Most of these settings relate to what I call the 'continuous' commands - those commands which keep running until stopped. That is, for example, the user says 'window move'...the window starts moving and continues until the user says 'window stop'. Also, a pop-up appears showing the status and how to stop it. Very similar to how 'wheel upper' and 'wheel stop' work.

I decided to expose the values governing these 'continuous' operations as settings because it's not clear how the behavior might differ between systems (mac, windows, linux) and various hardware configurations (system speed, multiple screens of differing sizes, etc).